Fix build error for special characters

rmagur1203 opened the pull request:
#63 

* Use ElementAccessExpression when accessing properties.
* When doing destructuring assignment, wrap the key in a StringLiteral.
* Normalizes characters that cannot be used in identifier.

GoogleFeud commented:
Thank you for the PR! It looks like you've changed the line endings of the file though, could you please use the line endings defined in the .prettierrc.json file?

jerzygangi commented:
@rmagur1203 Are you still planning to make the edits required on this PR?

rmagur1203 commented:
In prettierrc, the end of line is set to crlf, but in reality it seems to be set to lf.
Changed it to lf and modified it to merge into the `dev` branch.

GoogleFeud commented on src/gen/expressionUtils.ts:
```diff
@@ -171,7 +174,7 @@ export function _not(exp: ts.Expression): ts.Expression {
     if (typeof key === "string") {
         if (isInt(key)) return factory.createElementAccessExpression(exp, ts.factory.createNumericLiteral(key));
-        return ts.factory.createPropertyAccessExpression(exp, key);
+        return ts.factory.createElementAccessExpression(exp, ts.factory.createStringLiteral(key));
```
Wouldn't this always create element access expressions? It would be nice to create element access expressions only when necessary because otherwise code turns to this:

```ts
// We don't want this
errs["push"]("Expected value to be an object");
```

rmagur1203 replied:
It sure looks good to use createElementAccessExpression only when there are special characters, but since it is only used to access data and is a built file, it doesn't seem to matter much.
